instances.go

```diff
@@ -201,6 +234,55 @@ func (c *Client) SetAllowedNetworkInstancePolicy(ctx context.Context, enable boo
 	return err
 }
 
+// SetKeyAccessInstancePolicy updates the key create import access policy details associated with an instance.
+// For more information can refer to the Key Protect docs in the link below:
+// <kp link>
+func (c *Client) SetKeyAccessInstancePolicy(ctx context.Context, enable bool, attributes map[string]bool) error {
```

Comment:
Here is how I imagine we could call this and make it a bit easier for our API call as well

```
resp := SetKeyAccessInstancePolicy(
    AccessPolicyAttributes{
	   CreateRootKey: false,
	   ImportRootKey: true,
	}
)


# CreateStandardKey -> nil  ; # ignored for this request


{ 
   "createRootKey": false,
   "importRootKey": true,
}
```